domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ModifyWalletUrlsOnContractsToContainWalletApiTests.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using DataContext;
+using DataContext.Models;
+using DataContext.ValueObjects;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Testing.Helpers;
+using Xunit;
+
+namespace API.IntegrationTests.Shared.Migrations;
+
+[Collection(IntegrationTestCollection.CollectionName)]
+public class ModifyWalletUrlsOnContractsToContainWalletApiTests
+{
+    private readonly DbContextOptions<ApplicationDbContext> options;
+
+    public ModifyWalletUrlsOnContractsToContainWalletApiTests(IntegrationTestFixture integrationTestFixture)
+    {
+        var newDatabaseInfo = integrationTestFixture.PostgresContainer.CreateNewDatabase().Result;
```

Comment:
.Result <-- Could this cause problems on slow machines when not using .GetAwaiter().Result?

Reply:
we can't await in a constructor